14-11-2018/rafal/test_rot_13.rb

```diff
@@ -0,0 +1,28 @@
+require 'test/unit'
+require_relative "rot_13.rb"
+
+class TestRot13 < Test::Unit::TestCase
+  
+  def test_matches
+    assert_equal("uryyb", "hello".rot13)
+    assert_equal("urwany hejnal", "hejnal urwany".rot13)
+    assert_equal("hello", "hello".rot13.rot13)
+    assert_equal("uryyb", "HELLO".rot13)
+  end
+
+  def test_empty_string
+    assert_equal("", "".rot13)
+  end
+
+  def test_wrong_type
+    assert_raise(NoMethodError) {nil.rot13}    
```

Comment:
Po co to? Wiadomo ze 'nil' nie posiada tej metody, testujmy raczej funkcjonalność niż działanie rubiego.

Reply:
na slajdzie bylo